docker/gcp_pudl_etl.sh

```diff
@@ -62,9 +62,13 @@ function shutdown_vm() {
 }
 
 function copy_outputs_to_intake_bucket() {
-    echo "Copying outputs to intake bucket"
+    echo "Copying outputs to GCP intake bucket"
     gsutil -m -u $GCP_BILLING_PROJECT cp -r "$CONTAINER_PUDL_OUT/sqlite/*" "gs://intake.catalyst.coop/$GITHUB_REF"
     gsutil -m -u $GCP_BILLING_PROJECT cp -r "$CONTAINER_PUDL_OUT/parquet/epacems/*" "gs://intake.catalyst.coop/$GITHUB_REF"
+
+    echo "Copying outputs to AWS intake bucket"
+    aws s3 cp "$CONTAINER_PUDL_OUT/sqlite" "s3://intake.catalyst.coop/$GITHUB_REF" --recursive
+    aws s3 cp "$CONTAINER_PUDL_OUT/parquet/epacems" "s3://intake.catalyst.coop/$GITHUB_REF" --recursive
 }
```

Comment:
I would really like to change our local file layout to be flat, as it is in the buckets, for simplicity and so they can both be treated the same. Beyond this PR but just FYI...

Reply:
Oh so you'd like all of the ETL's outputs to live in one flat directory?

Reply:
I think so. They all have unique names, and we've started to comingle metadata outputs (JSON) with SQLite DB outputs. I just don't know that it adds anything to divide them up by filetype, and it does mean there's more path management to do.